decred/tests/unit/dcr/test_dcrdata_unit.py

```diff
@@ -170,20 +183,39 @@ def test_misc(self, http_get_post, capsys):
         out, err = capsys.readouterr()
         assert len(out.splitlines()) == 85
 
-    def test_subscriptions(self, http_get_post):
+    def test_subscriptions(self, http_get_post, monkeypatch):
+        # Create a DcrdataClient with a mocked ws.Client that captures the
+        # signals and stores the messages.
         preload_api_list(http_get_post)
-        ddc = DcrdataClient(BASE_URL)
+        ddc = TweakedDcrdataClient(BASE_URL, None, monkeypatch)
+        dcrdata._subcounter = 0
```

Comment:
Needed to get `request_id`s that are independent on the test execution order (boo globals, boo).